ui/src/components/policy-config.tsx

```diff
@@ -174,11 +174,13 @@ export function PolicyConfig() {
   }, []);
 
   const getAvailablePolicyTypes = (routeType: "http" | "tcp") => {
-    return Object.entries(POLICY_TYPES).filter(([_, info]) => {
-      if (routeType === "http") return !info.tcpOnly;
-      if (routeType === "tcp") return !info.httpOnly;
-      return true;
-    }).sort((a, b) => a[1].name.localeCompare(b[1].name));
+    return Object.entries(POLICY_TYPES)
```

Comment:
@peterj every UI PR seems to reformat things. Can we make a standard format command and require it in CI?

Reply:
FWIW, I tried bumping my npm version and still got a diff. Not sure if eslint config isn't being read or what

Reply:
its not just you, ~every PR seems to make a diff including my own